public/pages/DetectorResults/containers/AnomalyResultsTable.tsx

```diff
@@ -68,7 +74,14 @@ export function AnomalyResultsTable(props: AnomalyResultsTableProps) {
 
   useEffect(() => {
     const anomalies = props.anomalies
-      ? props.anomalies.filter(anomaly => anomaly.anomalyGrade > 0)
+      ? props.anomalies
+          .filter((anomaly) => anomaly.anomalyGrade > 0)
+          .map((anomaly) => {
+            return {
+              ...anomaly,
+              [ENTITY_VALUE_FIELD]: get(anomaly, 'entity[0].value'),
```

Comment:
To save space, maybe only add this extra entity field if `isHCDetector` flag is `true`? I'm thinking of the case where there may be a lot of anomalies for a non-HC detector where this field is unnecessary

Reply:
sure. will add such logic only for HC detector

Reply:
changed.